torchao/quantization/prototype/mixed_precision/scripts/BO_acc.py

```diff
@@ -0,0 +1,419 @@
+import sys
```

Comment:
should we call this BO_model_size.py to be consistent with the other script (BO_inference_speed.py)? Since both are optimizing for accuracy technically

Reply:
renamed to BO_acc_modelsize.py and BO_acc_throughput.py